core/logger/logger.go

```diff
@@ -211,10 +247,81 @@ func WithFile(file io.Writer) Option {
 	}
 }
 
+// WithFileRotation sets up the option to roll log files
+func WithFileRotation(prefix string, subfolder string, fileExtension string) Option {
+	return func(el *Logger) error {
+		file, err := newFile(prefix, subfolder, fileExtension)
+		if err != nil {
+			return err
+		}
+
+		el.roll = true
+		el.file.prefix = prefix
+		el.logFiles = append(el.logFiles, file)
+		el.file.SetWriter(file)
+		return nil
+	}
+}
+
+// WithStderrRedirect sets up the option to redirect stderr to file
+func WithStderrRedirect() Option {
+	return func(el *Logger) error {
+		file, err := newFile("fatalErrors"+el.file.prefix, "logs", "log")
```

Comment:
Should we maybe keep the prefix_fatalErrors pattern?

Reply:
Changed.